tests/pytests/test_json_multi_text.py

```diff
@@ -144,20 +144,53 @@ def testMultiTagString(env):
     conn.execute_command('JSON.SET', 'doc:4', '$', doc4_content)
 
     # Index multi flat values
-    env.expect('FT.CREATE', 'idx1', 'ON', 'JSON', 'SCHEMA', '$.category[*]', 'AS', 'category', 'TAG').ok()
+    env.expect('FT.CREATE', 'idx1', 'ON', 'JSON', 'SCHEMA',
+               '$.category[*]', 'AS', 'category', 'TAG').ok()
     # Index an array
-    env.expect('FT.CREATE', 'idx2', 'ON', 'JSON', 'SCHEMA', '$.category', 'AS', 'category', 'TAG').ok()
-    
+    env.expect('FT.CREATE', 'idx2', 'ON', 'JSON', 'SCHEMA',
+               '$.category', 'AS', 'category', 'TAG').ok()
+
     waitForIndex(env, 'idx1')
     waitForIndex(env, 'idx2')
-    
-    res1 = [1, 'doc:1', ['category', 'mathematics and computer science']]
-    res2 = [1, 'doc:1', ['category_arr', '["mathematics and computer science","logic","programming","database"]']]
-    
-    # Currently return a single value (only the first value)
```

Comment:
Why this test was changed?

Reply:
Previous behavior is still being tested (and was added another case with path $.category[1])
(see `# Test backward compatibility (before DIALECT 3)`)
And new behavior is also tested.